src/components/Card/Card.js

```diff
@@ -1,13 +1,57 @@
 import './Card.css'
 import createElement from '../../lib/createElement'
-import getColorByHouse from '../../services/getColorByHouse'
+// import getColorByHouse from '../../services/getColorByHouse'
```

Comment:
If you don't need this code anymore, then it's best to delete the comment before merging into main.